packages/flutter/test/cupertino/text_field_test.dart

```diff
@@ -3701,6 +3701,167 @@ void main() {
     );
   });
 
+  testWidgets(
+    'Can drag the left handle while the right handle remains off-screen',
+    (WidgetTester tester) async {
+      // Text is longer than textfield width.
+      const String testValue = 'aaaaaaaaaaaaaaaaaaaaaaaaaaa bbbbbbbbbbbbbbbbbbbbbbbbbbb';
+      final TextEditingController controller = TextEditingController(text: testValue);
+      final ScrollController scrollController = ScrollController();
+
+      await tester.pumpWidget(
+        CupertinoApp(
+          home: Center(
+            child: CupertinoTextField(
+              dragStartBehavior: DragStartBehavior.down,
+              controller: controller,
+              scrollController: scrollController,
+            ),
+          ),
+        ),
+      );
+
+      await tester.enterText(find.byType(CupertinoTextField), testValue);
+      await tester.pump();
+      await tester.pump(const Duration(milliseconds: 200));
+
+      // Double tap 'b' to show handles.
+      final Offset bPos = textOffsetToPosition(tester, testValue.indexOf('b'));
+      await tester.tapAt(bPos);
+      await tester.pump(const Duration(milliseconds: 50));
```

Comment:
Also here, could you replace this with pumpAndSettle?

Reply:
This too little time duration is for double tap so I think I need the duration.

Reply:
Ah I see.  Maybe use `kDoubleTapTimeout ~/ 2` instead?  Not a big deal, though.